bin/start_gke

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This is to be used by the Makefile for a start gke target.
+
+set -eo pipefail
+
+MY_IP=`curl icanhazip.com`
+echo "local ip is $MY_IP"
+
+USERNAME=`gcloud config get-value account | awk -F"@" '{print $1}'`
+CLUSTERNAME=$USERNAME-gke-sandbox
+
+# Create GKE Cluster - No TTL available, will need a external monitor with cleanup
+gcloud config set project moz-fx-data-gke-sandbox
+
+if gcloud container clusters describe $CLUSTERNAME --region us-west1 >/dev/null 2>&1; then
+    echo "cluster $CLUSTERNAME exists"
+else
+    echo "cluster $CLUSTERNAME doesn't exist. creating..."
+    gcloud container clusters create $CLUSTERNAME \
+           --enable-stackdriver-kubernetes \
+           -m n1-standard-4 \
+           --release-channel="stable" \
+           --enable-master-authorized-networks \
+           --master-authorized-networks="$MY_IP/32" \
+           --region us-west1 \
+           --num-nodes=1 \
+           --scopes="cloud-platform" \
+           --service-account="data-gke-sandbox-runner@moz-fx-data-gke-sandbox.iam.gserviceaccount.com"
+fi
+
+echo "fetching secret..."
+JSON_CREDS=`gcloud secrets versions access latest --secret="gke-sandbox-creds"`
```

Comment:
Looks like I missed this in the ops PR. Is there a reason we don't allow `roles/iam.serviceAccountKeyAdmin` and allow created keys directly instead of using a static secret? In theory we could create / destroy these at the same cadence as the GKE clusters themselves, but we need to consider cleanup steps as well I suppose.

Reply:
just for simplicity. Also, isnt granting view on secrets better than granting serviceAccountKeyAdmin?

Reply:
The `serviceAccountKeyAdmin` idiom is modeled after cloudops-deploylib and has the advantage that secrets are temporal (they're created per-run and deleted afterward). It requires proper cleanup though and we have an [open issue](https://github.com/mozilla-services/cloudops-infra-deploylib/issues/105) for that in deploylib land.

On the whole I'd prefer to not have static secrets that are susceptible to requiring rotation (when e.g. a DE leaves) but I'm also not strenuously objecting to it, especially when we haven't always had success implementing the temporal idiom in more secure environments.